src/main/java/net/openhft/chronicle/core/io/ReferenceCountedTracer.java

```diff
@@ -32,14 +32,14 @@
 public interface ReferenceCountedTracer extends ReferenceCounted {
 
     /**
-     * Factory method that creates a new instance of {@link ReferenceCountedTracer}. The instance returned
+     * Factory method that creates a new instance of . The instance returned
```

Comment:
This is missing a link

Reply:
It's not. I guess it's inherent reference to the enclosing type/class.
The space is extra though :)